modules/social_features/social_event/social_event.module

```diff
@@ -822,7 +888,8 @@ function social_event_entity_operation_alter(array &$operations, EntityInterface
   $node = \Drupal::routeMatch()->getParameter('node');
   // Check if the entity type is one of event_enrollment and that we're on the
   // correct view. Otherwise it would update all actions across the platform.
-  if ($entity->getEntityTypeId() === 'event_enrollment' && \Drupal::routeMatch()->getRouteName() === 'view.event_manage_enrollment_requests.page_manage_enrollment_requests') {
```

Comment:
@dmytriikaiun Looks like those changes not relevant and were added from code formatting of your IDE.